src/client.rs

```diff
@@ -2085,4 +2105,35 @@ mod test {
             .unwrap();
         assert_eq!(manifest.config.media_type, manifest::WASM_CONFIG_MEDIA_TYPE);
     }
+
+    #[tokio::test]
+    #[ignore]
+    async fn test_roundtrip_multiple_layers() {
+        tracing_subscriber::fmt::init();
+        let mut c = Client::new(ClientConfig {
+            protocol: ClientProtocol::HttpsExcept(vec!["oci.registry.local".to_string()]),
+            ..Default::default()
+        });
+        let src_image = Reference::try_from("registry:2.7.1").expect("failed to parse reference");
+        let dest_image = Reference::try_from("oci.registry.local/python:test")
```

Comment:
nitpicking, I think the target destination should be different

Reply:
ah yes, I've updated